tests/test_peer_did.py

```diff
@@ -1,13 +1,108 @@
 import pytest
 
+from peerdid.peer_did import create_peer_did_numalgo_0, create_peer_did_numalgo_2, resolve_peer_did
+from peerdid.types import PublicKeyAgreement, PublicKeyAuthentication, KeyTypeAgreement, KeyTypeAuthentication
 
-def test_create_numalgo_0():
-    pass
 
+def test_create_numalgo_0_positive():
+    signing_keys = [PublicKeyAuthentication(
```

Comment:
Any test need to have an assertion (check) at the end.
Here we may check the following:
```
assert peer_did_algo_0 == "<expected value>"
assert is_peer_did(peer_did_algo_0)
```